src/sentry/static/sentry/app/components/group/sidebar.jsx

```diff
@@ -301,5 +301,4 @@ const GroupSidebar = createReactClass({
   },
 });
 
-export {GroupSidebar};
-export default withEnvironment(GroupSidebar);
```

Comment:
Is this component imported elsewhere than in the groupEventDetails component?

Reply:
No, it's only used here, fyi 2 types of exports were for tests